fl4health/losses/contrastive_loss.py

```diff
@@ -86,3 +87,75 @@ def forward(
         labels = torch.zeros(features.size(0)).to(self.device).long()
 
         return self.cross_entropy_function(logits, labels)
+
+
+class NtXentLoss(nn.Module):
+    def __init__(self, device: torch.device, temperature: float = 0.05) -> None:
```

Comment:
Maybe this is a reasonable default (`0.05`) for the temperature, but it seems super low 😂